.github/workflows/docs-checks.yml

```diff
@@ -55,12 +59,17 @@ jobs:
           SPHINX_MOCK_REQUIREMENTS: 0
         working-directory: ./docs
         run: |
+          # ToDo: proper parametrize
           # First run the same pipeline as Read-The-Docs
           make doctest
           make coverage
```

Comment:
the `SOURCEDIR` should be specified to tell which docs to be built, otherwise the default is `source-pytorch` https://github.com/Lightning-AI/lightning/blob/b58577fd4db4acaddd2d02a039799697694a5901/docs/Makefile#L7

Reply:
I moved the file to the folder directly. This would be simpler.

Reply:
Thanks ! 
I'm not sure about having separate Makefiles for each app/pytorch. It's only a different param

Reply:
Same for the README: https://github.com/Lightning-AI/lightning/blob/ad87d2cad03520543713f83eec202566677cd128/docs/README.md

Ideally have only one README and the same command for building docs. e.g. `make docs pytorch` or `make docs app`

Reply:
I guess we need to call the python command directly and eventually remove make from the root until we have final unification...